### ISSUE TITLE
Missing Fluent UI Documentation

### ISSUE THREAD

**Issue by sydsutton** [microsoft/fluentui#25743]
### Describe the issue:

The documentation that used to be at "https://learn.microsoft.com/en-us/javascript/api/office-ui-fabric-react..." have been moved, and are nowhere to be found. If somebody could point me in the direction of where the docs have been moved to, that would be extremely helpful.

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Blocking

![image](https://user-images.githubusercontent.com/83406605/203158268-93389445-e37c-4bec-8f83-53ae31040183.png)

**Comment by ling1726** [comment]
Sorry I'm not aware of those docs, are they different from https://developer.microsoft.com/en-us/fluentui#/controls/web?

**Comment by sydsutton** [comment]
@ling1726 Yes, they are different. What I'm looking for, and what was there just probably 6 weeks ago, is documentation that has detailed information about Fluent UI "Interfaces", "Enums", and "Objects". I'm guessing it had over 1000+ links on one page, separated bt the categories I listed above.

**Comment by sydsutton** [comment]
The original link was "https://docs.microsoft.com/en-us/javascript/api/react?view=office-ui-fabric-react-latest" if that helps at all.

**Comment by sydsutton** [comment]
@ling1726 And here is the most recent archived version of the site https://web.archive.org/web/20220704002429/https://docs.microsoft.com/en-us/javascript/api/react?view=office-ui-fabric-react-latest

**Comment by micahgodbolt** [comment]
@sydsutton the docs that we had hosted within docs.microsoft.com were incredibly outdated, duplicative, and without a solid pipeline to update. We made the decision to remove those docs and focus on our primary source of documentation, which ling had already mentioned. 


The 1000+ list of enums and classes was created with an API extractor plugin. That same data can be found in our github repo (albeit in a pretty large file)

https://github.com/microsoft/fluentui/blob/7.0/packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md
